### ISSUE TITLE
bert performance 

### ISSUE THREAD

**Issue by flyoliver** [peidaqi/bert_gpu#1]
hi,  i finetune a classify model based on the original bert model and this repo. accuracy dropped 1%. parameters:
nohup python run_classifier.py   --task_name=sex   --do_train=true   --do_eval=true   --data_dir=**   --vocab_file=**  --bert_config_file=**   --init_checkpoint=**   --max_seq_length=128   --train_batch_size=16   --learning_rate=4e-5   --num_train_epochs=5   --output_dir=** --num_gpus=8 &
is my parameters wrong?
